webrender_traits/src/webgl.rs

```diff
@@ -73,6 +73,7 @@ pub enum WebGLCommand {
     GetAttribLocation(WebGLProgramId, String, MsgSender<Option<i32>>),
     GetUniformLocation(WebGLProgramId, String, MsgSender<Option<i32>>),
     GetVertexAttrib(u32, u32, MsgSender<WebGLResult<WebGLParameter>>),
+    GetVertexAttribOffset(u32, u32, MsgSender<WebGLResult<isize>>),
```

Comment:
I'm slightly concerned about the naming here, given that `GetVertexAttribOffset` translates to `get_vertex_attrib_pointer_v` call. I fully agree that the semantics is much better with `*Offset`, but is it fine to (i.e. is this the only case of) have the WR webgl command name not exactly matching the underlying GL call?